### DISCUSSION TITLE
Performant lyapunov based safe RL method

### DISCUSSION THREAD

**Discussion by hari-sikchi** [PKU-Alignment/omnisafe#241] [Ideas]
Hi,

Congratulations on creating a great hub for safe RL methods. Another algorithm that might be interesting for the Lyapunov category is:
Paper: https://arxiv.org/abs/2103.09230
Code: https://github.com/hari-sikchi/LBPO

**Comment by Gaiejj** [comment]
Thank you very much for your affirmation! OmniSafe will consider incorporating safe reinforcement learning algorithms based on the Lyapunov method into our future support list and provide documentation on them. Do you have any more specific and technical suggestions that could help us further improve this work?
